lstchain/io/io.py

```diff
@@ -1065,3 +1069,72 @@ def read_dl2_to_pyirf(filename):
         events[k] *= v
 
     return events, pyirf_simu_info
+
+
+def read_dl2_sub_run(t_filename, columns_to_read=None):
```

Comment:
This function is not limited to sub runs.
So I'd rename it `read_dl2_params` so it can be used more broadly.
Also update the docstring accordingly, it reads only the parameters.